pages/index.tsx

```diff
@@ -0,0 +1,560 @@
+          <b>Data managers</b>
+        </div>
+        <div style={{ display: 'flex' }}>
+          <b>Developers</b>
+        </div>
+      </div>
+      <div className={styles.row}>
+        <PageLink
+          image={AnalyticsManagerIcon}
+          title="Set up webhook publishing"
+          callToAction={
+            new CallToAction(
+              'Get started',
+              '/workspace/tracking-plan/publishing#a-namewebhooka-webhook',
+            )
+          }
+        />
+        <PageLink
+          image={DevIcon}
+          title="Listen to the tracking plan changes with a webhook"
+          callToAction={
+            new CallToAction(
+              'Get started',
+              '/implementation/avo-tracking-plan-webhook',
+            )
+          }
+        />
+      </div>
+      <div className={styles.row}>
+        <PageLink
+          image={AnalyticsManagerIcon}
+          title="Use Inspector to monitor analytics implementation"
+          callToAction={
+            new CallToAction(
+              'Get started',
+              '/data-design/start-using-inspector',
+            )
+          }
+        />
+        <PageLink
+          image={DevIcon}
+          title="How does Avo Inspector SDK work?"
+          callToAction={
+            new CallToAction(
+              'Get started',
+              '/implementation/devs-101#avo-inspector-sdk',
+            )
+          }
+        />
+      </div>
+      <div className={styles.row}>
+        <PageLink
+          image={AnalyticsManagerIcon}
+          title="Using Mobile Debuggers"
+          callToAction={
+            new CallToAction(
+              'Get started',
+              '/explore-tracking-plan/start-using-visual-debuggers#accessing-mobile-debuggers',
+            )
+          }
+        />
+        <PageLink
+          image={DevIcon}
+          title="Avo Inspector SDK reference"
+          callToAction={
+            new CallToAction('Read', '/implementation/inspector/sdk/')
+          }
+        />
+      </div>
+      <div className={styles.row}>
+        {emptyRowElement()}
+        <PageLink
+          image={DevIcon}
+          title="Set up the Mobile Debuggers"
+          callToAction={
+            new CallToAction('Read', '/implementation/setup-mobile-debugger')
+          }
+        />
+      </div>
+
+      <h2 id="stage3">3. Better tracking implementation</h2>
+
+      <p className={styles.text}>
+        Avo generates human readable, type-safe, tracking code called Avo Functions, that sends the data exactly as planned. No more data bugs or cross-platform discrepancies in your analytics tracking!
+      </p>
+
+      <p className={styles.text}>
+        This step requires developers to adopt Avo into their tracking implementation workflow.
+      </p>
+      <p className={styles.text}>
+        It&apos;s not necessary to swap all your tracking with Avo functions,
+        you can start with a single tracking call.
```

Comment:
It's not necessary to swap all your tracking with Avo functions, in fact we recommend starting by only using Avo functions for new events.